aws/ec2/snappy_ec2.py

```diff
@@ -331,6 +333,16 @@ def parse_args():
     parser.add_option(
         "--instance-profile-name", default=None,
         help="IAM profile name to launch instances under")
+    parser.add_option(
+        "--assume-role-arn", type="string", default=None,
+        help="ARN of assumed role to launch a instance")
```

Comment:
A more elaborate help would help. e..g "ARN of the IAM role to assume. This IAM role's credentials will be used to launch the cluster."